### ISSUE TITLE
Request: Accessibility updates for screen readers

### ISSUE THREAD

**Issue by vikraman-alea-bah** [react-component/progress#304]
This package is not accessible to screen readers. 

If you use a screen reader and select the progress bar it announces as "image" instead of the percentage of the progress bar. 

I would suggest allowing for an attached label to give the progress bar an accessible name, as well as separately allowing the value to be announced to a screen reader. 

You can test with screen readers for free using VoiceOver on a Mac or NVDA on a Windows computer.
